javaparser-core/src/main/java/com/github/javaparser/javadoc/Javadoc.java

```diff
@@ -56,6 +56,16 @@ public Javadoc addBlockTag(String tagName, String content) {
         return addBlockTag(new JavadocBlockTag(tagName, content));
     }
 
+    /**
+     * for tags like "@param abc this is a parameter" where
```

Comment:
Capital F for `for` (oh my god, how much time I have to waste to write this pointless comments :D)